deployment/gem.rb

```diff
@@ -192,6 +192,29 @@ def ask(question, default)
     end
   end
 
+  namespace :redirect do
+    desc "Enable a redirect page, all traffic will be redirect to this page."
+    task :enable do
+      production_url = ask("What is the production url?", domain).gsub('http://','').gsub('https://', '')
```

Comment:
zet hij automatisch een spatie achter je zin in de ask method? Anders zou ik nog een spatie plaatsen achter het vraagteken.